### DISCUSSION TITLE
Is it possible to have the font changed on websites which Dark Reader is configured not to apply the dark theme?

### DISCUSSION THREAD

**Discussion by RaptaG** [darkreader/darkreader#13512] [Q&A] [answered]
I'd like to keep the dark theme of some sites but change the font for all

**Comment by Myshor** [comment, marked as answer]
Not using Dark Reader. The font settings is applied only with all other Dark Reader settings related to changing theme.
So if you have some sited excluded from applying theme then the font won't be aplied to them.

If you want to have font changed on all sites then separate extension/addon should be used. Look for "Font changer" as I was able to saw it as first tip after searching for some using google. 😉

BTW... but if you would use some separate extension then be sure to have option to change font in Dark Reader disabled because injecting this change from two different extensions can cause some strange issues and problems.
